mri/operators/fourier/cartesian.py

```diff
@@ -23,21 +23,21 @@
 
 
 class FFT(OperatorBase):
-    """ Standard unitary ND Fast Fourrier Transform class.
-    The FFT will be normalized in a symmetric way
+    """ Standard unitary ND Fast Fourier Transform (FFT) class.
```

Comment:
(D210) No whitespaces allowed surrounding docstring text
```suggestion
    """Standard unitary ND Fast Fourier Transform (FFT) class.
```